modules/apps/company/company-test/src/testIntegration/java/com/liferay/company/service/test/CompanyLocalServiceTest.java

```diff
@@ -277,7 +276,7 @@ public void testAddAndDeleteCompanyWithDLFileEntryTypes() throws Exception {
 
 		DLFileEntryType dlFileEntryType =
 			_dlFileEntryTypeLocalService.getFileEntryType(
-				DLFileEntryTypeConstants.FILE_ENTRY_TYPE_ID_BASIC_DOCUMENT);
+				company.getGroupId(), "GOOGLE_DOCS");
```

Comment:
@yuhai I do not understand why you changed the logic here, please keep the logic as before if it does not block you

Reply:
@tinatian, because the test method doesn't include any **Assert**. I remember there is a checker to block this if the test method didn't include **Assert**